tests/testerina-integration-test/src/test/resources/command-outputs/unix/BasicCasesTest-testAnnotations.txt

```diff
@@ -4,6 +4,9 @@ Compiling source
 Running Tests with Coverage
 
 	annotations
+
+		Test execution time :*****ms
```

Comment:
I think we must print the time at the end. When the list is big, this might go unnoticed.